R/any_duplicated_linter.R

```diff
@@ -35,7 +35,7 @@
 #' @export
 any_duplicated_linter <- function() {
```

Comment:
Should the docs and examples be expanded as well?

Reply:
Sure, good idea. I actually am not sure what a "canonical" (non-contrived) example using `n_distinct()` would look like TBH, so omitted for now.